### PR TITLE
feat: Reports database config

### PR DESCRIPTION
**Pull request by strongishllama**
- Started the reports page, completed the setup section and added headings for writing and running reports
- Updated doc_menu.yml module items to be capitalised

refs: 8396

### CODE REVIEW

File: _documentation/core_modules/report/report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+type: doc
+---
+
+## Introduction
+
+The report module allows users to write custom MYSQL reports that can be saved and run when needed.
+
+## Setup
+
+1. Before you learn how to use reports you'll need to make your config is setup correctly. Add the following entries into the <b>config.php</b> file in the root directory of your Cmfive project.
```
Comment (careck) [inline _documentation/core_modules/report/report.md]: missing word: "you'll need to make **sure** your config"

File: _documentation/core_modules/report/report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Config::set("report.database", [
+    "hostname"  => "DB_HOSTNAME",
+    "username"  => "DB_USERNAME",
+    "password"  => "DB_PASSWORD",
+    "database"  => "DB_NAME",
+    "driver"    => "mysql"
+]);
+```
+2. Replace <b>DB_HOSTNAME</b>, <b>DB_USERNAME</b>, <b>DB_PASSWORD</b> and <b>DB_NAME</b> with their respective values.
+3. Make sure to clear your config cache to apply your changes.
```
Comment (careck) [inline _documentation/core_modules/report/report.md]: too wordy: remove "make sure" from the beginning of the sentence

**Comment by strongishllama** [comment]
@careck Implemented request changes.
